### PR TITLE
[1.13] Fix missing image names on list

### PR DESCRIPTION
**Pull request by saschagrunert**
Backport of #2455

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
/approve

**Comment by haircommander** [comment]
you're getting lint errors:
` make lint
checking lint
server/image_list.go:42:1:warning: exported function ConvertImage should have comment or be unexported (golint)
Makefile:74: recipe for target 'lint' failed
make: *** [lint] Error 1
The command "make lint" exited with 2.`

**Comment by haircommander** [comment]
otherwise LGTM

**Comment by saschagrunert** [comment]
> you're getting lint errors:
> ` make lint checking lint server/image_list.go:42:1:warning: exported function ConvertImage should have comment or be unexported (golint) Makefile:74: recipe for target 'lint' failed make: *** [lint] Error 1 The command "make lint" exited with 2.`

Hm, I added a comment so this should be now fixed here.

**Comment by saschagrunert** [comment]
CircleCI won't be happy since there exists no build config within that branch.

File: server/image_list.go [hunk 1]
```diff
@@ -30,22 +31,45 @@ func (s *Server) ListImages(ctx context.Context, req *pb.ListImagesRequest) (res
...
+
+	repoTags := []string{"<none>:<none>"}
+	if len(from.RepoTags) > 0 {
+		repoTags = from.RepoTags
+	}
+
+	repoDigests := []string{"<none>@<none>"}
+	if len(from.RepoDigests) > 0 {
+		repoDigests = from.RepoDigests
+	}
```
Comment (mtrmac) [inline server/image_list.go]: I can’t see how this makes sense: https://github.com/cri-o/cri-o/pull/2455#discussion_r291677114 .

**Comment by haircommander** [comment]
manually merging because there is no build config...

**Pull request merged**